diff --git a/setup.py b/setup.py
index 1f223a7..2780270 100755
--- a/setup.py
+++ b/setup.py
@@ -140,7 +140,7 @@ def get_carolina_extension():
     
      
     external_libs = ['boost_regex', 'boost_filesystem', 'boost_serialization',
-                     'boost_system', 'boost_signals', boost_python]
+                     'boost_system', 'boost_program_options', boost_python]
 
     print(boost_python)
 
