diff --color -Naur a/CMakeLists.txt b/CMakeLists.txt
--- a/CMakeLists.txt	2025-02-19 15:23:16
+++ b/CMakeLists.txt	2025-02-19 15:26:28
@@ -165,10 +165,19 @@
 # Perl is required for examples, docs, and system tests
 find_package(Perl REQUIRED)
 
+set(CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} -Wall -lpthread -lutil")
 # Python is optionally required by Dakota, Teuchos, and Acro; probe
 # for components here at top-level:
+include_directories(${PYTHON_INCLUDE_DIRS})
+include_directories(/usr/include)
+include_directories(/usr/lib64)
+include_directories(/tmp/INSTALL_DIR/dakota-6.21.0-public-src-cli/packages/external/eigen3/include/)
+find_package(Threads REQUIRED)
+set (PYBIND11_PYTHON_VERSION "3.11")
+
 include(DakotaFindPython)
 dakota_find_python()
+add_link_options()
 
 # Conditionally find Java JDK needed for input spec, docs, Java API
 include(DakotaFindJava)
@@ -208,6 +217,9 @@
         CACHE STRING "fix boost::serialization" FORCE)
   endif()
 endif()
+
+link_directories(/usr/lib64)
+link_directories(/tmp/INSTALL_DIR/lib)
 
 include(DakotaFindSystemTPLs)
 # Unconditionally find BLAS/LAPACK or equivalent
