--- a/cmake/DakotaFindPython.cmake	2025-06-05 15:32:21
+++ b/cmake/DakotaFindPython.cmake	2025-06-05 15:33:45
@@ -22,7 +22,7 @@
 
     endif()
 
-    find_package(Python3 REQUIRED ${dakota_python_components})
+    find_package(Python3 COMPONENTS ${dakota_python_components} REQUIRED)
 
     if (DAKOTA_PYTHON_DIRECT_INTERFACE_NUMPY)
 	message(STATUS "NumPy version ${Python_NumPy_VERSION} found at ${Python_NumPy_INCLUDE_DIRS}")
