--- a/cmake/DakotaFindPython.cmake	2023-11-24 10:57:08
+++ b/cmake/DakotaFindPython.cmake	2023-11-24 11:18:52
@@ -17,7 +17,7 @@
 
     endif()
 
-    find_package(Python REQUIRED ${dakota_python_components})
+    find_package(Python COMPONENTS ${dakota_python_components} REQUIRED)
 
     # pybind11, C3, Acro, etc., use older CMake FindPythonInterp, so we
     # coerce it to use same as Dakota; more complex situations may
